static/app/views/performance/vitalDetail/vitalChart.tsx

```diff
@@ -140,7 +140,7 @@ function VitalChart({
 
                 const seriesMax = getMaxOfSeries(smoothedSeries);
                 const yAxisMax = Math.max(seriesMax, vitalPoor);
-                chartOptions.yAxis.max = yAxisMax * 1.1;
+                chartOptions.yAxis!.max = yAxisMax * 1.1;
```

Comment:
is there any way to avoid having to use `!`?

Reply:
we know it will always exist but the type says yAxis could be undefined, could wrap it but i think we'd want it to error if it doesn't exist instead of just stop working